install_and_upgrade.sh

```diff
@@ -71,6 +71,11 @@ else
   die "  Please update your config/settings.yml file to accommodate the above errors"
 fi
 
+log_running "Installing MeteorDemod and OpenCV as its dependency"
+sudo apt install python3-opencv
+sudo apt install ~/raspberry-noaa-v2/software/Meteordemod-2.0.0-Linux.deb
```

Comment:
These packages should be installed by ansible in this file: https://github.com/jekhokie/raspberry-noaa-v2/blob/master/ansible/roles/common/tasks/dependencies.yml